crate/azure_sev_quote/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+use std::str::FromStr;
+
+use tss_esapi::{
+    abstraction::nv, handles::NvIndexTpmHandle, interface_types::resource_handles::NvAuth,
+    tcti_ldr::TctiNameConf, Context,
+};
+
+use crate::error::Error;
+
+pub mod error;
+
+const AZURE_QUOTE_NVINDEX: u32 = 0x1400001;
```

Comment:
is it clearer?
```suggestion
const AZURE_QUOTE_NVINDEX: u32 = 0x0140_0001;
```
 discard comment otherwise

Reply:
I prefer to keep the same format as `tpm2_nvreadpublic` command.